chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/assaults/MemoryAssaultIntegrationTest.java

```diff
@@ -22,7 +20,6 @@
 /**
  * @author Benjamin Wilms
  */
-@RunWith(SpringRunner.class)
```

Comment:
Glad it works without that. I thought it would need `@ExtendWith(SpringExtension.class)` or something similar